src/shared/components/VideoPreview/VideoPreview.styles.tsx

```diff
@@ -56,7 +57,29 @@ export const CoverIcon = styled(Icon)`
   color: ${colors.white};
 `
 
+export const RemoveButton = styled(Button)`
+  position: absolute;
+  top: ${sizes(2)};
+  right: ${sizes(2)};
+  width: 39px;
+  height: 40px;
+  background-color: ${colors.blue};
+  color: ${colors.white};
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+  cursor: pointer;
+  ${CoverIcon} {
+    width: 20px;
+    height: 20px;
+    transform: translateY(0);
```

Comment:
Why use `CoverIcon` as styling base? Because of that we need to override its styles which don't have anything to do with this specific icon.  If you use just `Icon` instead, we probably will only need to add `color` and won't need to override `transform`